openmc/checkvalue.py

```diff
@@ -1,3 +1,17 @@
+from collections import Iterable
+from numbers import Integral
+
+import numpy as np
+
+def _isinstance(value, expected_type):
```

Comment:
In this case I think it's worth the tradeoff. I agree it would be good to include some commentary explaining why the code is there in the first place.